webrtc.html

```diff
@@ -3153,6 +3155,25 @@ <h2>Methods</h2>
                     </ol>
                   </li>
                 </ol>
+                <p>
+                  A candidate is <dfn>administratively prohibited</dfn> if the
+                  UA has decided not to allow connection attempts to this
+                  address.
+                  The UA MUST prohibit connections to addresses on the
+                  [[!Fetch]] [= block bad port =] list, and MAY choose
+                  to not make connections to other addresses.
+                </p>
+                <p>
+                  If the {{RTCConfiguration/iceTransportPolicy}} member of the {{RTCConfiguration}} is {{RTCIceTransportPolicy/relay}}, candidates requiring external
+                  resolution, such as mDNS candidates and DNS candidates,
+                  MUST be prohibited.
+                </p>
+                <p>
+                  For privacy
+                  reasons, there is no indication to the user about whether
+                  or not an address/port is blocked; it behaves exactly as
+                  if there was no response from the address.
```

Comment:
Move this para to definition.

Reply:
I wouldn't use "user" here (which would typically be used for the end-user, not the developer); Fetch uses "developer", so I guess we could re-use that language here.

Reply:
the developer might still be able to figure this out based on lack of candidate pair stats (which is what bit us re tcp) or lack of remote stats?

Reply:
It's OK to let the developer learn whether or not there was a TURN server that accepted his credentials, or a peer that accepted his ufrag/pw, at that address.
What's not OK is to let the developer know the difference between a blocked port and a port where nobody's home. I think that's what this is trying to say.